services/management/dbaas/components_service.go

```diff
@@ -458,3 +462,27 @@ func (c componentsService) InstallOperator(ctx context.Context, req *dbaasv1beta
 
 	return &dbaasv1beta1.InstallOperatorResponse{Status: dbaasv1beta1.OperatorsStatus_OPERATORS_STATUS_OK}, nil
 }
+
+// LatestRecommended returns the latest recommended version from a list of product versions or, if there are no
+// recommended versions, returns the latest.
```

Comment:
Wrong description. As I see in case if there is no `default` component this function returns error, right?

Reply:
probably we can rename this function to DefaultComponent